components/automate-cli/cmd/chef-automate/certRotate.go

```diff
@@ -213,6 +216,13 @@ func (c *certRotateFlow) certRotate(cmd *cobra.Command, args []string, flagsObj
 			return errors.New("Error occured while fetching current certs.")
 		}
 
+		if flagsObj.timeout < DEFAULT_TIMEOUT {
+			return errors.Errorf("The operation timeout duration for each individual node during the certificate rotation process should be set to a value greater than %v seconds.", DEFAULT_TIMEOUT)
+		}
+
+		sshConfig.timeout = flagsObj.timeout
```

Comment:
do the Implementation of timeout in the ssh command execution part of any other PR?

Reply:
Yes